packages/payload/src/config/types.ts

```diff
@@ -880,6 +906,10 @@ export type Config = {
      * @default 'all' // The theme can be configured by users
      */
     theme?: 'all' | 'dark' | 'light'
+    /**
+     * Configure timezone related settings for the admin panel.
+     */
+    timezone?: TimezoneConfig
```

Comment:
Feels as though this should be plural:
```suggestion
    timezones?: TimezoneConfig
```